test/rcdnispecs.js

```diff
@@ -0,0 +1,39 @@
+describe('Testing rcdni directive',function(){
+	var scope,										
+      	directive,
+      	compiled,
+      	html,
+      	ctrl,
+      	form;
+
+    beforeEach(function(){						
+    	module('app.directives');
+      module('app.services');					
+    	html = '<form name="patientForm">' +		
+    			'<input type=text id=dni name="dni" rc-dni="" ng-model="fields.dni" />'
```

Comment:
Miguel is right. It is necessary to add to concatenate the string because at the end of the line quotes are closed.